### PR TITLE
Remove reliance on `enc-` prefix when decrypting secret files

### PR DESCRIPTION
**Pull request by sgibson91**
This was identified as redundant in https://github.com/2i2c-org/infrastructure/pull/1098#issuecomment-1067869844

### CODE REVIEW

File: deployer/file_acquisition.py [hunk 1]
```diff
@@ -143,8 +141,8 @@ def get_decrypted_file(original_filepath):
             yield f.name
 
     else:
-        # For a file that does not match our naming conventions for secrets, yield the
-        # original path
+        # The file does not have "secret" in it's name, therefore does not need to be
```
Comment (sgibson91) [inline deployer/file_acquisition.py:144]:
```suggestion
        # The file does not have "secret" in its name, therefore does not need to be
```

**Comment by yuvipanda** [comment]
@sgibson91 based on https://github.com/2i2c-org/infrastructure/pull/1098#issuecomment-1067885768, if you feel strongly about keeping `enc-` we can keep it!

**Comment by sgibson91** [comment]
@yuvipanda I think the logic here is actually separate to our discussion on naming conventions. If we're looking for "secret" in the name, then we pick up all files, whether or not they have the `enc-` prefix. The important bit is to raise an error if we don't find a `sops` key in the content, as that points to leaked credentials.

**Comment by yuvipanda** [comment]
Ah right - you can have a secret file that isn't encrypted and doesn't have enc- in your local checkout, but this code only hits things that are actually read by the deployer, and hence likely to be committed! makes sense.

**Review by yuvipanda — APPROVED** [review]

**Comment by yuvipanda** [comment]
Thanks for explaining it to me, @sgibson91!

**Pull request merged**